cudapoa/src/cudapoa_structs.cuh

```diff
@@ -45,6 +49,11 @@
 #define CUDAPOA_KERNEL_ERROR_ENCOUNTERED UINT8_MAX
 #define CUDAPOA_KERNEL_NOERROR_ENCOUNTERED 0
 
+// NW return error codes
+#define NW_BACKTRACKING_LOOP_FAILED -1
```

Comment:
let's add the `CUDAPOA_KERNEL` prefix to each of these macros

Reply:
done.